Python_Backend/dummy_server.py

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/env python
+
+import asyncio
+import base64
+import binascii
+import io
+import numpy
+from PIL import Image, UnidentifiedImageError
+import websockets
+import contextvars
+
+import AnswerGenerator
+import DFGenerator
+from StateGenerator import StateGenerator
```

Comment:
These unused imports can be removed from the dummy server as they are not being used.

Reply:
imports have been cleaned up